app.js

```diff
@@ -1,30 +1,41 @@
 const express = require('express');
 const app = express();
-const { getTopics, getApi} = require("./controll/controller")
+const { getTopics, getApi, getArticleById} = require("./controll/controller")
 
+app.use(express.json())
```

Comment:
This line of code isn't needed until we make a post request. Please be careful not to add code extra to the ticket